src/main.ts

```diff
@@ -397,6 +397,36 @@ async function getWorkspaceDependency(
 	outputChannel: OutputChannel,
 ): Promise<string | undefined> {
 	for (const workspaceFolder of workspace.workspaceFolders ?? []) {
+		// Check for Yarn PnP and try resolving the Biome binary without a node_modules
+		// folder first.
+		if (await fileExists(Uri.joinPath(workspaceFolder.uri, ".pnp.cjs"))) {
```

Comment:
Looks like yarn may also use the `.js` extension.

Just tried locally with

```shell
yarn install --pnp
```

Reply:
What version of Yarn is this? I can't seem to find documentation of a `.pnp.js` file or the `--pnp` option anywhere?

Reply:
```shell
yarn --version
1.22.21
```

Reply:
Just realizing this version is old AF. Not sure if its even supported.

Reply:
Holy hell, if didn't realize they added PnP support to the "classic" versions, too. I suggest, not supporting this old version, please have at a minimal repo using Yarn 4 here: https://gist.github.com/roblillack/5818b0dbb5bfd69757bc4397ff6f8b32

Reply:
Is this not used anymore? Looks like `1.22.21` was released 3 months ago. It also happened to be the default version I got when using corepack.

Reply:
Ok, it seems the API is 100% compatible for our use-case. I added support for `.pnp.js`, too.